mitodl/mitxpro
# asadiqbal08/command unblock_users to remove users from the blocklist.

#### Pre-Flight checklist


#### What are the relevant tickets?
fixes: #2229 

#### What's this PR do?
As an xPRO administrator, I want the ability to unblock a user from the blocklist to allow them to register.

#### How should this be manually tested?
- First, Block a user with `retire_users` management command, Run this with `-b` option.
- You can see an entry in the blocklist model.
- User will not be able to create an account with blocked email while visiting a create account page in xPro.
- Now, run this command for unblocking the user.
- User will be able to use the earlier blocked email while visiting a create account page in xPro.
